### PR TITLE
docs: Update API tuts

### PR DESCRIPTION
**Pull request by adela-bytebase**

### CODE REVIEW

File: content/docs/tutorials/api.md [hunk 1]
```diff
@@ -94,7 +94,7 @@ repo](https://github.com/adela-bytebase/bb-api-sample) and clone it.
 8. Go back to the sample application, and click **Refresh status**. You'll see it's from [OPEN] to [DONE].
 ![sample-done](/content/docs/tutorials/api/sample-done.webp)
 
-1. For additional information about the sample application, refer to the README in its corresponding [GitHub repository](https://github.com/adela-bytebase/bb-api-sample/).
+1. For additional information about the sample application, refer to the README in its corresponding [GitHub repository](https://github.com/bytebase/bb-api-sample/).
```
Comment (boojack) [inline content/docs/tutorials/api.md:97]:
```suggestion
9. For additional information about the sample application, refer to the README in its corresponding [GitHub repository](https://github.com/bytebase/bb-api-sample/).
```

**Review by boojack — APPROVED** [review]
LGTM

**Review by tianzhou — APPROVED** [review]
LGTM

File: content/docs/tutorials/api.md (hunk 1 above)
Comment (tianzhou) [reply content/docs/tutorials/api.md:97]: I will create a separate PR to fix all these

**Pull request merged**